doc/user/user/pipelines/index.md

```diff
@@ -337,11 +341,8 @@ For example, you may encounter an error when running the "Assembly and Annotatio
 
 ![analyses-table-error](images/analyses-table-error.png)
 
-If you click on the <span class="fa fa-fw fa-question-circle"></span> icon, you can preview the Galaxy job error information:
-
-![analyses-table-error-preview](images/analyses-table-error-preview.png)
 
-If you follow the link to the analysis page, you can view all the job error information for the pipeline in error:
+If you click on the analysis name it will take you to the analysis page. From here you can view all the job error information for the pipeline including the galaxy job information, the galaxy parameters, standard error, and standard output.
 
 ![analysis-job-error](images/analysis-job-error.png)
```

Comment:
This image should be redone with a scaled down browser.  It's too big to see what it's trying to show.

Reply:
Updated in 6d265b032635f8e0f76f98eddb88a30e899bf36f